cmds/modprobe/modprobe.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// modprobe - Add and remove modules from the Linux Kernel
+//
+// Synopsis:
+//     modprobe modulename...
+//
+// Author:
+//     Roland Kammerer <dev.rck@gmail.com>
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/u-root/u-root/pkg/kmodule"
+	"golang.org/x/sys/unix"
+)
+
+const cmd = "modprobe modulename"
+
+type modState uint8
+
+const (
+	_ modState = iota
+	loading
+	loaded
+)
+
+type dependency struct {
+	state modState
+	deps  []string
+}
+
+type depMap map[string]*dependency
```

Comment:
leave a comment for what this map is?

// depMap is a map of module path -> dependency list.

Reply:
OK